apps/storefront-e2e/src/support/page_fragments/header.fragment.ts

```diff
@@ -34,30 +34,39 @@ export class HeaderFragment {
   };
 
   changeLocale = (locale: string) => {
-    cy.intercept({
-      method: 'GET',
-      url: /(\/catalog-search.*|\/concrete-products\/.*)/,
-    }).as('productOrSearchRequests');
-    // hydrate
-    this.getLocaleButton().click();
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(500);
-    this.getLocaleButton().click();
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    this.initProductsUpdateInterceptor();
+
+    this.getLocaleButton().trigger('mouseenter').wait(75).click();
```

Comment:
I'd probably move this `75` somewhere to utility layer, so to have a constant for ui delay.

Reply:
I'm not sure that this number is consistent and will be used somewhere else(
It's more like a temporary solution for hydration, not a final one...